sympy/parsing/autolev/_listener_autolev_antlr.py

```diff
@@ -737,18 +737,18 @@ def exitFunction(self, ctx):
                 expr = self.getValue(ch.expr(0))
                 if ch.expr(0) in self.matrix_expr or (expr in self.type.keys() and self.type[expr] == "matrix"):
                     self.matrix_expr.append(ctx)
-                    self.setValue(ctx, "sm.Matrix([sm.factor(i, " + self.getValue(ch.expr(1)) + ") for i in " +
+                    self.setValue(ctx, "_sm.Matrix([_sm.factor(i, " + self.getValue(ch.expr(1)) + ") for i in " +
                                   expr + "])" + ".reshape((" + expr + ").shape[0], " + "(" + expr + ").shape[1])")
                 else:
-                    self.setValue(ctx, "sm.factor(" + "(" + expr + ")" +
+                    self.setValue(ctx, "_sm.factor(" + "(" + expr + ")" +
                                   ", " + self.getValue(ch.expr(1)) + ")")
 
             # D(y, x)
-            elif func_name == "d":
+            elif func_name == "_d":
```

Comment:
Change this back. The others should be `"_d"`.

Reply:
Sure @NikhilPappu